src/Components/ArticlePage.js

```diff
@@ -0,0 +1,98 @@
+import "./ArticlePage.css";
+import { useParams } from "react-router-dom";
+import {
+  getArticleById,
+  getArticleCommentsById,
+  patchArticleVote,
+} from "../Api/Api";
+import React, { useEffect, useState } from "react";
+import Comment from "./Comment";
+import thumbsDown from "../Assets/thumbs-down-outline.svg";
+import thumbsUp from "../Assets/thumbs-up-outline.svg";
+
+const ArticlePage = () => {
+  const { id } = useParams();
+
+  const [articleComments, setArticleComments] = useState([]);
+
+  const [article, setArticle] = useState(null);
+
+  // fetches articles if none have been passed through state or props using the id from params to obtain correct article
+
+  useEffect(
+    () => {
+      if (!article) {
+        const fetchArticles = async () => {
+          try {
+            const response = await getArticleById(id);
+            setArticle(response.data.articleById);
+          } catch (error) {
+            console.log(error);
+          }
+        };
+        fetchArticles();
+      }
+    },
+    [id],
+    [article]
+  );
+
+  useEffect(() => {
+    if (article) {
+      const fetchComments = async () => {
+        try {
+          const response = await getArticleCommentsById(id);
+          setArticleComments(response.data.articleComments);
+        } catch (error) {
+          console.log(error);
+        }
+      };
+      fetchComments();
+    }
+  }, [id, article]);
+
```

Comment:
I would separate the comments part into it's own component. This is to reduce strain on one single component so you don't have multiple useEffect's in one component. You can then have one useEffect in each component and then render the comments component on your article page.